tox.ini

```diff
@@ -99,6 +103,11 @@ changedir=
   update_cluster: {toxinidir}/tests/functional/centos/7/cluster
   bluestore_journal_collocation: {toxinidir}/tests/functional/centos/7/bs-journal-collocation
   bluestore_cluster: {toxinidir}/tests/functional/centos/7/bs-cluster
+  bluestore_dmcrypt_journal: {toxinidir}/tests/functional/centos/7/bs-dmcrypt-dedicated-journal/
+  bluestore_dmcrypt_journal_collocation: {toxinidir}/tests/functional/centos/7/bs-dmcrypt-journal-collocation/
+  bluestore_docker_cluster: {toxinidir}/tests/functional/centos/7/bs-docker-cluster
+  bluestore_docker_dedicated_journal: {toxinidir}/tests/functional/centos/7/bs-docker-cluster-dedicated-journal
+  bluestore_docker_dmcrypt_journal_collocation: {toxinidir}/tests/functional/centos/7/bs-docker-cluster-dmcrypt-journal-collocation
```

Comment:
`tests/functional/centos/7/bs-docker-cluster-dmcrypt-journal-collocation` doesn't exist in your PR